runtime-testsuite/resources/org/antlr/v4/test/runtime/descriptors/CompositeParsers/BringInLiteralsFromDelegate.txt

```diff
@@ -18,6 +18,4 @@ s
 =a
 
 [output]
-"""S.a
-"""
-
+"""S.a"""
```

Comment:
I'm nervous about changing the output. You must have changed the semantics of this.  Sometimes the \n is part of the output.

Reply:
In this case we have action `a : '=' 'a' {<write("\"S.a\"")>};` that definitely should not print new line. That's why I think it's correct change. Also, there is inconsistent code across runtimes: some of the them prints new lines, some of them are not. The code that fixes new lines stuff is place in separated commit: https://github.com/antlr/antlr4/pull/3750/commits/08f5ae7098ad084e9abfa60f3b74d14e656e2118

Reply:
From a quick scan it looks like you have always added a new line across runtimes? Then just stripped?

Reply:
Trailing new line was always being added to the output: https://github.com/antlr/antlr4/blob/dev/runtime-testsuite/test/org/antlr/v4/test/runtime/TestOutputReading.java#L35 I rewrote the behavior and now all new lines are being handled accurately: https://github.com/KvanTTT/antlr4/blob/runtime-tests-refactoring-2/runtime-testsuite/test/org/antlr/v4/test/runtime/StreamReader.java#L40 (it only removes `\r` chars to unify output across platforms).

Reply:
ok. cool.